src/storage-client/src/client.proto

```diff
@@ -89,6 +89,7 @@ message ProtoStorageResponse {
         uint64 bytes_received = 7;
         uint64 envelope_state_bytes = 8;
         uint64 envelope_state_count = 9;
```

Comment:
im surprised I left these as non-optional...

Reply:
I think that might have been me 😅